fcrepo-search-impl/src/main/java/org/fcrepo/search/impl/DbSearchIndexImpl.java

```diff
@@ -840,7 +837,7 @@ private void cleanupTransaction(final String txId) {
     private List<String> toggleForeignKeyChecks(final boolean enable) {
 
         if (isPostgres()) {
-            return EMPTY_LIST;
+            return (List<String>)EMPTY_LIST;
```

Comment:
You could just make this `return Collections.emptyList();`

Reply:
will do,.